dictionaries/en_shared/src/exclude-words.txt

```diff
@@ -0,0 +1,4 @@
+~colum
+colum
+knifes
```

Comment:
where does knifes comes from ?

Reply:
maybe moving dictionaries/en_US/src/exclude-words.txt to dictionaries/en_shared/src/exclude-words.txt instead of dictionaries/en_GB/src/exclude-words.txt

so you would have kept the historical info ?

Reply:
A couple of the Hunspell dictionaries incorrectly generate `knifes` and `midwifes`. I try not to change the Hunspell source.